Docs: Add Redshift batch exports (#7840)
PostHog/posthog.com: contents/docs/cdp/batch-exports/redshift.md

3. Click "Create export workflow".
4. Select **Redshift** as the batch export type.
5. Fill in the necessary [configuration details](#redshift-configuration).
6. Finalize the creation by clicking on "Create".
7. Done! The batch export will schedule its first run on the start of the next period.

## Redshift configuration

Configuring a batch export targeting Redshift requires the following Redshift-specific configuration values:

- **User:** A Redshift user name with permissions to insert data into the specified table and, if the table does not exist, permissions to create the table.

- **Password:** The password for the Redshift user specified.

- **Host:** The endpoint of your Redshift cluster, excluding the port number and database name.

- **Port:** The port number on which the Redshift cluster is listening (default is 5439).

- **Database:** The name of the Redshift database to which the data is to be exported.

- **Schema:** The name of the schema within the database. This determines where the table for exporting data will be located.

- **Table name:** The name of the table where the data will be inserted.

> - **Properties data type:** The data type configurations for the properties column in the table,either `VARCHAR(65535)` or `SUPER`.

Reviewer: This affects all JSON like fields, so `properties`, `set`, and `set_once`. Also, the value should be just `varchar`, without the length (i.e. 65535).

Reviewer: ```suggestion
- **Properties data type:** The data type configurations for the properties, set, and set_once columns, either `VARCHAR(65535)` or `SUPER`.
```

Reviewer: Feel free to edit or omit it!

- **Events to exclude:** A list of events to omit from the exported data.

> - **Events to include:** A list of events to include from the exported data. If added, only these events will be exported.

Reviewer: ```suggestion
- **Events to include:** A list of events to include in the exported data. If added, only these events will be exported.
```
